.github/pull_request_template.md

```diff
@@ -1,12 +1,22 @@
-Thanks for creating a PR! Please follow this template and delete items/sections that are not relevant to your changes, including these instructions.
+# Pull Request Instructions
 
-Please also make sure your [PR title](https://github.com/pinterest/gestalt/#releasing) matches our format: `{ComponentName}: Description (mentioning platform if relevant)`, or `{ComponentName}, {OtherComponentName}: Description (mentioning platform if relevant)` if multiple components are affected.
+Thanks for creating a PR! 🎉
+
+- Please follow this template and delete items/sections that are not relevant to your changes, _including these instructions_.
+- Make sure your [PR title](https://github.com/pinterest/gestalt/#releasing) matches our format: `{ComponentName}: Description (mention platform if relevant)`. If there are changes to multiple components, use `{ComponentName}, {OtherComponentName}: Description (mention platform if relevant)`.
+- Each PR needs a [label](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) indicating which [semantic version](https://semver.org/) should be applied:
```

Comment:
If there are other common (or tricky) changes that should be called out here, please suggest them!